html/janus.js

```diff
@@ -22,185 +22,39 @@
 	OTHER DEALINGS IN THE SOFTWARE.
  */
 
-// List of sessions
-Janus.sessions = {};
-
-// Screensharing Chrome Extension ID
-Janus.extensionId = "hapfgfdkleiggjjpfpenajgdnfckjpaj";
-Janus.isExtensionEnabled = function() {
-	if(window.navigator.userAgent.match('Chrome')) {
-		var chromever = parseInt(window.navigator.userAgent.match(/Chrome\/(.*) /)[1], 10);
-		var maxver = 33;
-		if(window.navigator.userAgent.match('Linux'))
-			maxver = 35;	// "known" crash in chrome 34 and 35 on linux
-		if(chromever >= 26 && chromever <= maxver) {
-			// Older versions of Chrome don't support this extension-based approach, so lie
-			return true;
-		}
-		return ($('#janus-extension-installed').length > 0);
+/**
+ * UMD wrapper.
+ */
+(function (root, factory) {
+	if (typeof define === "function" && define.amd) {
+		define([
+			"adapter",
+			"jquery",
+		], function(adapter, jQuery) {
```

Comment:
This stuff is not needed anymore, we don't load external JavaScript files programmatically. Both are linked externally in html files, and currently point to their cdnjs link.